src/ember-app.js

```diff
@@ -180,43 +185,28 @@ class EmberApp {
   }
 
   /**
-   * @private
-   *
-   * Loads the app and vendor files in the sandbox (Node vm).
-   *
+   * Perform any cleanup that is needed
    */
-  loadAppFiles() {
-    let sandbox = this.sandbox;
-    let appFilePaths = this.appFilePaths;
-    let vendorFilePaths = this.vendorFilePaths;
-
-    sandbox.eval('sourceMapSupport.install(Error);');
-
-    debug('evaluating app and vendor files');
-
-    vendorFilePaths.forEach(function(vendorFilePath) {
-      debug('evaluating vendor file %s', vendorFilePath);
-      let vendorFile = fs.readFileSync(vendorFilePath, 'utf8');
-      sandbox.eval(vendorFile, vendorFilePath);
-    });
-    debug('vendor file evaluated');
-
-    appFilePaths.forEach(function(appFilePath) {
-      debug('evaluating app file %s', appFilePath);
-      let appFile = fs.readFileSync(appFilePath, 'utf8');
-      sandbox.eval(appFile, appFilePath);
-    });
-    debug('app files evaluated');
-  }
+  destroy() {}
```

Comment:
I think the `reload` function still calls this method: https://github.com/ember-fastboot/fastboot/blob/master/src/index.js#L93, should we just remove it?

Reply:
I don't want to remove it yet, because I want to make it do something in a follow up PR (but before v3.0.0). Specifically, I want to "warm" the cache even more which means I'll have to be able to cleanup some of the async.

Reply:
aah ok can we add a todo then here?

Reply:
Yep! Will do.